apps/evalite-ui/app/routes/__root.tsx

```diff
@@ -154,15 +114,16 @@ const EvalSidebarItem = (props: {
 
   return (
     <SidebarMenuItem key={props.name}>
-      <NavLink
-        prefetch="intent"
-        to={`/eval/${props.name}`}
-        className={({ isActive }) =>
-          cn(
-            "flex justify-between text-sm px-2 py-1 rounded hover:bg-gray-100 transition-colors",
-            isActive && "bg-gray-200 text-gray-800 hover:bg-gray-200"
-          )
+      <Link
+        preload="intent"
+        to={`/eval/$name`}
+        params={{ name: props.name }}
+        className={
+          "flex justify-between text-sm px-2 py-1 rounded hover:bg-gray-100 transition-colors"
         }
+        activeProps={{
```

Comment:
`activeProps` will merge it's classname with the base classname. I kinda wanna find a way to customize the merger to use the `cn` util, but practically I've not seen issues with it so far.